tools/GraphClust/AlignCluster/align_cluster.xml

```diff
@@ -102,12 +102,12 @@
           <when value="yes" >
               <param format="fasta" name="transcript_loci_fasta" type="data" label="Loci reference sequence"
                 help="Sequence of reference species (human) for the genomic loci (precursor mRNA/lncRNA) to locate the clusters" />
-              <param name='transcript_loci_bed' type="text" value="chr1 0 100000 gene 0 +" 
+              <param name='transcript_loci_bed' type="txt" value="chr1 0 100000 gene 0 +" 
                 help="bed entry string of the reference transcript loci for bed/ucsc-track output. 'chrom start end name score strand' white-space separated.">
                 <validator type="regex" message="one-line bed string must have at least 6 entries as BED6 format (chrom start end name 0 strand) 
                 (space and tab allowed)">^\S+\s+[0-9]+\s+[0-9]+\s+\S+\s+\S+\s+[-+]\s*$</validator> 
               </param>
-              <param name='genome_version' value="hg38" type="text" label="reference genome assembly"
+              <param name='genome_version' value="hg38" type="txt" label="reference genome assembly"
```

Comment:
ah shit, @pavanvidem this was wrong. a parameter can hve the type="text"

A data-prameter can only have the format "txt"

Reply:
Sorry, my mistake. I accidentally replaced all `text` to `txt`. I will revert it